File: SDKs/yodlee.com/1.1.0/go/pkg/models/shared/evaluateaccountaddress.go
package shared

type EvaluateAccountAddressTypeEnum string

const (
	EvaluateAccountAddressTypeEnumHome             EvaluateAccountAddressTypeEnum = "HOME"
	EvaluateAccountAddressTypeEnumBusiness         EvaluateAccountAddressTypeEnum = "BUSINESS"
	EvaluateAccountAddressTypeEnumPobox            EvaluateAccountAddressTypeEnum = "POBOX"
	EvaluateAccountAddressTypeEnumRetail           EvaluateAccountAddressTypeEnum = "RETAIL"
	EvaluateAccountAddressTypeEnumOffice           EvaluateAccountAddressTypeEnum = "OFFICE"
	EvaluateAccountAddressTypeEnumSmallBusiness    EvaluateAccountAddressTypeEnum = "SMALL_BUSINESS"
	EvaluateAccountAddressTypeEnumCommunication    EvaluateAccountAddressTypeEnum = "COMMUNICATION"
	EvaluateAccountAddressTypeEnumPermanent        EvaluateAccountAddressTypeEnum = "PERMANENT"
	EvaluateAccountAddressTypeEnumStatementAddress EvaluateAccountAddressTypeEnum = "STATEMENT_ADDRESS"
	EvaluateAccountAddressTypeEnumPayment          EvaluateAccountAddressTypeEnum = "PAYMENT"
	EvaluateAccountAddressTypeEnumPayoff           EvaluateAccountAddressTypeEnum = "PAYOFF"
	EvaluateAccountAddressTypeEnumUnknown          EvaluateAccountAddressTypeEnum = "UNKNOWN"
)

type EvaluateAccountAddress struct {
	Address1   *string                         `json:"address1,omitempty"`
	Address2   *string                         `json:"address2,omitempty"`
	Address3   *string                         `json:"address3,omitempty"`
	City       *string                         `json:"city,omitempty"`
	Country    *string                         `json:"country,omitempty"`
	SourceType *string                         `json:"sourceType,omitempty"`
	State      *string                         `json:"state,omitempty"`
	Street     string                          `json:"street"`
	Type       *EvaluateAccountAddressTypeEnum `json:"type,omitempty"`
	Zip        *string                         `json:"zip,omitempty"`
}

File: SDKs/yodlee.com/1.1.0/java/lib/src/main/java/openapisdk/models/shared/EvaluateAccountAddress.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

public class EvaluateAccountAddress {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("address1")
    public String address1;
    public EvaluateAccountAddress withAddress1(String address1) {
        this.address1 = address1;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("address2")
    public String address2;
    public EvaluateAccountAddress withAddress2(String address2) {
        this.address2 = address2;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("address3")
    public String address3;
    public EvaluateAccountAddress withAddress3(String address3) {
        this.address3 = address3;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("city")
    public String city;
    public EvaluateAccountAddress withCity(String city) {
        this.city = city;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("country")
    public String country;
    public EvaluateAccountAddress withCountry(String country) {
        this.country = country;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("sourceType")
    public String sourceType;
    public EvaluateAccountAddress withSourceType(String sourceType) {
        this.sourceType = sourceType;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("state")
    public String state;
    public EvaluateAccountAddress withState(String state) {
        this.state = state;
        return this;
    }
    @JsonProperty("street")
    public String street;
    public EvaluateAccountAddress withStreet(String street) {
        this.street = street;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("type")
    public EvaluateAccountAddressTypeEnum type;
    public EvaluateAccountAddress withType(EvaluateAccountAddressTypeEnum type) {
        this.type = type;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("zip")
    public String zip;
    public EvaluateAccountAddress withZip(String zip) {
        this.zip = zip;
        return this;
    }
}